### ISSUE TITLE
some easy to fix small errors

### ISSUE THREAD

**Issue by danceb** [gis-ops/docker-valhalla#98]
the current image `nn-valhalla-pr-4055` names `"transit_feeds_dir": "/gtfs_dir"` in line 143, but it should be `"transit_feeds_dir": "/gtfs_feeds"`.

If the `transit_files` dir is already created and `build_transit=Force` is set, I am getting the error `"rm: cannot remove '/custom_files/transit_tiles': Is a directory"`.

**Comment by nilsnolde** [comment]
> "transit_feeds_dir": "/gtfs_dir" in line 143 in the valhalla.json

Hm, you're sure that you're running the right image @danceb ? I ran it earlier, and a `cat valhalla.json| grep transit` printed `"transit_feeds_dir": "/gtfs_feeds"`.

The other problem is rooted here: https://github.com/gis-ops/docker-valhalla/blob/cb221387fefd8be4150c665d80e1e1a5eab66596/scripts/configure_valhalla.sh#L39. Missing a `-r` :smile: It doesn't matter really though, Valhalla'll remove that folder in any case and start from scratch. But maybe we'll change that in the future though, so I put that rm line in our image. I'll fix that real quick.

**Comment by danceb** [comment]
Is there a parameter which has to be "True" or something else to I have to take care of to get the valhalla.json file generated? I have deleted my image and the config file und pulled the image again. Now the valhalla.json file is generated but as an emtpy file ...

My config looks like this:

```
- serve_tiles=True
- use_tiles_ignore_pbf=True
- tileset_name=valhalla_tiles
- force_rebuild=False
- build_elevation=False
- build_admins=False
- build_time_zones=False
- build_transit=False 
- build_tar=False
```

**Comment by nilsnolde** [comment]
Yeah I'm sorry, there was apparently yet another tiny bug.. I fixed it both in master and the branch, before I pushed the branch to re-build the image. And master is finally testing fine again.

Can you re-pull the latest `nn-valhalla-pr-4055` and try again after removing any previous files? 

No idea how it worked for me this morning tbh.. Sorry for the hassle :sweat_smile:

**Comment by nilsnolde** [comment]
Uff, good to hear:)
